docs/readmes/lte/sessiond.md

```diff
@@ -80,3 +80,72 @@ On SessionD restart, session state will be synced to MME service, which
 it will use as the source of truth for active sessions. 
 Based on the active sessions, PipelineD will be instructed to 
 allow/disallow traffic.
+
+## Enforcement algorithm 
+Sessiond can report credit to PCRF (Gx) and OCS (Gy). In both cases it is up to 
+PCRF or OCS to determine when the user is out of quota. Sessiond will only receive, 
+count, report and execute actions from the core. The algorithms here described 
+are mainly related how to interpret the orders received by PCRF and OCS and how
+in advance we will report to the core to prevent to hit the exhaustion prematurely.
+Sessiond will not execute any action unless receiving an explicit order from 
+PCRF and OCS.
+
+Gx reporting will be identified with monitoring key, while Gy reporting will be 
+identified by charging key. 
+
+Both Gx and Gy reporting have some commonalities on its algorithm:
+1. PCRF/OCS will send an initial grant with allowed credit to be reported. 
+2. Sessiond will count the credits used but it will not report back anything yet
+3. When a specific percentage of that grant (by default 80%) is reached, sessiond
```

Comment:
Might be good to mention the config value for this

Reply:
Yes I will add the name of the setting, but the value (80%) i am considering changing it